frontend/src/pages/loser.html

```diff
@@ -0,0 +1,17 @@
+<html>
+
+<head>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css">
+	<link rel="stylesheet" href="../css/loser.css">
```

Comment:
Hocam rip olan ben değilimdir inş sjsnks

Reply:
Yok hocam bu sefer kendimi gömdüm